OpenSim/Simulation/Wrap/WrapCylinder.cpp

```diff
@@ -800,3 +801,23 @@ bool WrapCylinder::_adjust_tangent_point(SimTK::Vec3& pt1,
 
     return did_adust;
 }
+
+// Implement generateDecorations by WrapCylinder to replace the previous out of place implementation 
+// in ModelVisualizer
+void WrapCylinder::generateDecorations(bool fixed, const ModelDisplayHints& hints, const SimTK::State& state,
+    SimTK::Array_<SimTK::DecorativeGeometry>& appendToThis) const 
+{
+    
+    if (hints.get_show_wrap_geometry()) {
+        const Vec3 color(SimTK::Cyan);
+        SimTK::Transform ztoy;
+        ztoy.updR().setRotationFromAngleAboutX(SimTK_PI / 2);
```

Comment:
Would be good to add a comment for why this Rotation offset is necessary, otherwise it seems tempting to get rid of it.